src/main/java/com/plivo/api/models/powerpack/PowerpackResponse.java

```diff
@@ -1,8 +1,52 @@
 package com.plivo.api.models.powerpack;
 
 import com.plivo.api.models.base.BaseResponse;
-
 public class PowerpackResponse extends BaseResponse {
 
+  private String createdOn;
+  private String name;
+  private String application_type;
+  private String application_id;
+  private Boolean sticky_sender;
+  private Boolean local_connect;
+  private String number_pool;
+  private String uuid;
+  public NumberPriority[] number_priority;
+
+  public String getCreatedOn() {
+    return createdOn;
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public String getApplication_type() {
```

Comment:
The underscore doesn't conform to method naming conventions of Java:
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Reply:
Done